syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -822,3 +825,16 @@ def combine(self, *pointers):
         ps.append(self)
 
         return syft.combine_pointers(*ps)
+
+    def keep(self, *args, **kwargs):
```

Comment:
What does keep do? Could you add a docstring/explain why this is needed for native tensor

Reply:
It keeps the child promise with the argument given.
I need it to be able to call the method on wrapped promise but thinking about it, I'm not 100% sure it's the right way to do it.
(This is the same for value.)